src/components/ResetPasswordConfirm.tsx

```diff
@@ -0,0 +1,86 @@
+// ResetPasswordConfirm component
+import React, { useState } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import axios from 'axios';
+import Alert from './Alert';
+import logo from "../assets/logo.png";
+
+const VITE_BACKEND_URL = 'https://dev-kosj-api.fly.dev'; // Your backend URL
+
+const ConfirmResetPassword = () => {
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (password !== confirmPassword) {
+      setErrorMessage("Passwords do not match!");
+      return;
+    }
+    try {
+      const token = searchParams.get('token'); // Or use useParams for dynamic route parameter
```

Comment:
token should be outside, if there is no token, we can redirect to another page (using useEffect maybe)